39.md

```diff
@@ -0,0 +1,63 @@
+NIP-39
+======
+
+External Identities in Profiles
+-------------------------------
+
+`draft` `optional` `author:pseudozach` `author:Semisol`
+
+## Abstract
+
+Nostr protocol users may have other online identities such as usernames, profile pages, keypairs etc. they control and they may want to include this data in their profile metadata so clients can parse, validate and display this information.
+
+## `i` tag on a metadata event
+
+A new optional `i` tag is introduced for `kind 0` metadata event contents in addition to name, about, picture fields as included in [NIP-01](https://github.com/nostr-protocol/nips/blob/master/01.md):
+```json
+{
+    "id": "123",
+    "pubkey": "xxx",
+    ...
+    "tags": [
+        ["i", "github:semisol", "9721ce4ee4fceb91c9711ca2a6c9a5ab"],
+        ["i", "twitter:semisol_public", "1619358434134196225"],
+        ["i", "mastodon:bitcoinhackers.org/@semisol", "109775066355589974"]
+        ["i", "telegram:1087295469", "nostrdirectory/770"]
+    ]
+}
+```
+
+An `i` tag will have two parameters, which are defined as the following:
+1. `identity`: This is the identity provider name (for example `github`) and the identity  (for example `semisol`) joined together.
```

Comment:
You later refer to `ìdentity` but mean `semisol` not `github:semisol` and `identity` is specified twice in this line.

```suggestion
1. `platform:identity`: This is the platform name (for example `github`) and the identity on that platform (for example `semisol`) joined together with `:`.
```

Reply:
I agree. @Semisol any objections to this?